components/infra-proxy-service/Makefile

```diff
@@ -0,0 +1,47 @@
+include ../../Makefile.common_go
+
+PACKAGE_PATH = github.com/chef/automate/components/infra-proxy-service
+
+BINS = ${PACKAGE_PATH}/cmd/infra-proxy-service
+MIGRATION_READMES = storage/postgres/migration/sql/README.md
+
+packages:=${PACKAGE_PATH}/...
+ifdef CI
+  verbose?="-v"
+endif
+
+
+all: lint build test
+
+static: lint ${MIGRATION_READMES}
+unit: build test
+
+.PHONY: ${MIGRATION_READMES}
+${MIGRATION_READMES}:
+	../../scripts/generate_and_check_migration_files.sh $@
+
+# Regenerate all *.pb.go files
+proto:
```

Comment:
I don't think this is needed because all the protos for the infra-proxy-service are in the api folder.

Reply:
Somehow it left untouched, have removed it thanks